makefiles/tools/dfu.inc.mk

```diff
@@ -2,7 +2,14 @@ DFU ?= dfu-util
 FLASHER ?= $(DFU)
 HEXFILE = $(BINFILE)
 
-FFLAGS ?= -D $(HEXFILE) --reset $(DFU_ARGS)
+ifdef ROM_OFFSET
+  FLASH_ADDR = $(shell printf "0x%x" $$(($(ROM_START_ADDR) + $(ROM_OFFSET))))
+else
+  FLASH_ADDR = $(ROM_START_ADDR)
```

Comment:
This whole condition could be redone in a declarative way
```
_ROM_ADDR_WITH_OFFSET ?= $(shell printf "0x%x" $$(($(ROM_START_ADDR) + $(ROM_OFFSET))))
FLASH_ADDR ?= $(if $(ROM_OFFSET),$(_ROM_ADDR_WITH_OFFSET),$(ROM_START_ADDR))
```

It has the same output as the current PR when printing the flash command.